### ISSUE TITLE
Where is landmark_detect_best.pth

### ISSUE THREAD

**Issue by khang-nguyen2907** [open-mmlab/mmfashion#136]
Hi, I want to run the **test_landmark_detector.py** (I run on google colab) but I don't know where can I get "landmark_detect_best.pth". I go to MODEL_ZOO.md, download the pretrained model of Fashion Landmark Detection with vgg16 but the file I get is "latest.pth". Then I use it to run the demo on an image. It just turns out 3 coordinates of landmark. My output is: 

pretrained model checkpoint/vgg16.pth
Use load_from_local loader
The model and loaded state dict do not match exactly

unexpected key in source state_dict: classifier.0.weight, classifier.0.bias, classifier.3.weight, classifier.3.bias, classifier.6.weight, classifier.6.bias

model built
Use load_from_local loader
load checkpoint from: /content/mmfashion/checkpoint/latest.pth
/usr/local/lib/python3.7/dist-packages/torch/nn/functional.py:1351: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
detected landmark 77.60672433035714 93.52818080357142
detected landmark 144.5418962751116 91.87636021205357
detected landmark 89.90164620535714 93.12472098214285

Could anybody help me?

**Comment by xinzi2018** [comment]
me too

**Comment by waterworld8** [comment]
Same. I tried the same and got "The model and loaded state dict do not match exactly" and other warning messages. The detected landmarks are all near the top-left corner, which are obviously wrong:

detected landmark 23.393638857773375 6.335447622197016
detected landmark 8.05462385926928 19.54633380685534
detected landmark 27.81294988734382 16.453559058053155
detected landmark 12.884916365146639 25.944502013070245
detected landmark 24.20463732310704 25.090379374367853
detected landmark 10.798835967268264 32.082101276942666
detected landmark 27.47284578425544 33.402897630419055

I've spent several hours on this repo and I think I'm done with it.
